yrs/src/lib.rs

```diff
@@ -115,9 +115,9 @@ pub struct ClientBlockList {
     pub integrated_len: usize,
 }
 
-
 pub struct BlockStore {
     pub clients: HashMap<u64, ClientBlockList, BuildHasherDefault<ClientHasher>>,
+    pub local_block_list: ClientBlockList,
```

Comment:
I was just about to get rid of `local_block_list`. The idea was that the local client would not have to access `clients.get(local_client_id)` to get to the blocklist. But it turned out that this produces too many cases.

Reply:
No need to get rid of it now as long as it compiles. But maybe you could add a `todo` notice that we want to remove this.